### ISSUE TITLE
Latest Release Breaks SVN Integration

### ISSUE THREAD

**Issue by phila213** [microsoft/vscode#123623]
Issue Type: <b>Bug</b>

After installing the latest release the SVN icons are no longer apearing. There are 'placeholders' where the SVN actions icons were located in the source control tab, and the tooltips still appear when the mouse is moved over the placeholders, but the actions no longer work.

VS Code version: Code 1.56.0 (cfa2e218100323074ac1948c885448fdf4de2a7f, 2021-05-04T22:09:06.405Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz (8 x 3600)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.92GB (26.37GB free)|
|Process Argv|--crash-reporter-id 26071ec8-b4c3-4805-9c4c-f96d0b3105c6|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (13)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.0.4
project-manager|ale|12.1.0
clipboard-history|Anj|1.0.7
vscode-intelephense-client|bme|1.7.1
pgformatter|bra|1.17.0
bracket-pair-colorizer-2|Coe|0.2.0
xml|Dot|2.5.1
prettier-vscode|esb|6.3.2
svn-scm|joh|2.13.5
sqltools|mtx|0.23.0
vscode-thunder-client|ran|1.2.2
php-formatter|Sop|0.2.4
vscode-icons|vsc|11.4.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt602:30294772
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30291489
pythontb:30283811
pythonvspyt551:30291414
vspre833cf:30267465
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vscorecescf:30290706
pythondataviewer:30285071
vscus158cf:30286554

```

</details>

**Comment by gjsjohnmurray** [comment]
1.56.1 just shipped. Please update and see if your issue remains.

**Comment by gjsjohnmurray** [comment]
Update: it seems 1.56.1 was a security update. The fix I think you're looking for will be in 1.56.2

**Comment by joaomoreno** [comment]
Can you reproduce this in the [insider release](https://code.visualstudio.com/insiders)?

**Comment by phila213** [comment]
Hi,This is now fixed in v1.56.2Cheers, PhilSent from my Galaxy
-------- Original message --------From: João Moreno ***@***.***> Date: 17/5/21  04:04  (GMT+10:00) To: microsoft/vscode ***@***.***> Cc: phila213 ***@***.***>, Author ***@***.***> Subject: Re: [microsoft/vscode] Latest Release Breaks SVN Integration (#123623) 
Can you reproduce this in the insider release?
